rollup.config.js

```diff
@@ -190,7 +190,7 @@ export function makeBaseBundleConfig(options) {
       esModule: false,
     },
     plugins: [
-      jsVersion === 'es5' ? typescriptPluginES5 : typescriptPluginES6,
+      jsVersion.toLowerCase() === 'es5' ? typescriptPluginES5 : typescriptPluginES6,
```

Comment:
Why not change this to strict equals against `ES5`? We do this anyway here https://github.com/getsentry/sentry-javascript/blob/57a82f37aaad1f3e5c802b4b8055d71aa49d721e/packages/integrations/rollup.config.js#L15

Note, after we convert `buildBundles` into a node script, we can extract the constants so they are used everywhere (instead of referring to standalone strings)

Reply:
I had the choice to switch it to uppercase everywhere, or make it case-insensitive. I figured the latter was better because then it doesn't matter what case we use, which prevents this problem from happening in the future.

(You're correct that if/once we use a constant, it will be a moot point.)